backend/src/main/java/WebApi.java

```diff
@@ -35,21 +38,21 @@ public static void main(String[] args) {
     private static void setupCustomerInterface() {
         // Customers
         get("/customers", (request, response) -> ci.getAllCustomers(), gson::toJson);
-        get("/customers/:customerID", ((request, response) -> ci.getCustomer(request.params(":customerID"))), gson::toJson);
-        get("/customers/:customerID/orders,", ((request, response) -> ci.getOrdersFromCustomer(request.params(":customerID"))), gson::toJson);
-        get("/customers/:customerID/digitalparts,", ((request, response) -> ci.getDigitalPartsFromCustomer(request.params(":customerID"))), gson::toJson);
-        get("/customers/:customerID/physicalparts,", ((request, response) -> ci.getPhysicalPartsFromCustomer(request.params(":customerID"))), gson::toJson);
+        get("/customers/:customerID", ((request, response) -> ci.getCustomer(request.params("customerID"))), gson::toJson);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "customerID" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)